### ISSUE TITLE
Most elements should not be selectable and right-clickable

### ISSUE THREAD

**Issue by tomchen** [arcomage/arcomage-hd#60]
- [x] Nothing should be selectable except \<input\>s and text in "Help" window
- [x] Nothing should be right-clickable except \<input\>s and text in "Help" window
- [x] In Multiplayer Mode, user shouldn't be able to click "Connect" button with empty opponent id
- [x] A click where mousedown inside window & mouseup outside window shouldn't close the window; tab select a button, press enter to open the window then re-press enter, the window should close; auxclick outside the window should close the window
